### PR TITLE
Drop checklist

### PR DESCRIPTION
**Pull request by MathieuGilbert**
We're no longer using the changelog (one of the checklist items), and not using the other checklist items anyways, so dropping this functionality. This will also make it more obvious when people forget PR messages. 
- [ ] Change log
- [ ] Demo page
- [ ] Product owner signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,61 +0,0 @@
...
-[#29]: https://github.com/amaabca/lita-github_pr_list/issues/29
-[#30]: https://github.com/amaabca/lita-github_pr_list/issues/30
-[#32]: https://github.com/amaabca/lita-github_pr_list/issues/32
-[#33]: https://github.com/amaabca/lita-github_pr_list/issues/33
-[#35]: https://github.com/amaabca/lita-github_pr_list/issues/35
-[#37]: https://github.com/amaabca/lita-github_pr_list/issues/37
-[#38]: https://github.com/amaabca/lita-github_pr_list/issues/38
-[@MathieuGilbert]: https://github.com/MathieuGilbert
-[@darkodosenovic]: https://github.com/darkodosenovic
-[@rubene]: https://github.com/rubene
```
Comment (adamrmelnyk) [inline CHANGELOG.md]: Yay

**Comment by adamrmelnyk** [comment]
:book:

**Comment by adamrmelnyk** [comment]
:elephant: :elephant: :elephant:

**Comment by ryanjones** [comment]
Is there a way to remove all of the current hooks?

**Comment by MathieuGilbert** [comment]
@ryanjones calling lita remove then add hooks

**Pull request merged**